src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,159 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+
+function Pagination({ pageContext, type, path }) {
+  let { currentPage, totalPages, minPage, maxPage } = pageContext
+  let eachPage = 0
+
+  const apiPageChange = value => {
+    if (value !== currentPage) pageContext.formData(pageContext.formResponse, value)
+  }
+
+  const maxPageFn = count => {
+    if (count === 'More') {
```

Comment:
Is count a number or a string?

Reply:
count wasn't a very good name. Renamed to clicked because it represents the innerText of the pagination element clicked.